recipe/meta.yaml

```diff
@@ -16,6 +16,7 @@ source:
   {{ hash_type }}: {{ hash }}
 
 build:
+  noarch: python
   entry_points:
     - dw=datadotworld.cli:cli
```

Comment:
As per upstream issue ( https://github.com/conda/conda-build/issues/2378 ), please add spaces on either side of `=`.